test/ngAnimate/animateSpec.js

```diff
@@ -934,6 +935,195 @@ describe('animations', function() {
       }));
     });
 
+
+    describe('$animate.cancel()', function() {
```

Comment:
Isn't it enough to check that `runner.cancel` is called, since `runner.cancel` appears to have comprehensive tests of its own?

Reply:
Hm, that's right, just the runner tests don't test with actual DOM elements and not with the $animate functions. Your call whether it should be simplified.